pipelines/hacbs/hacbs-test.yaml

```diff
@@ -37,6 +37,22 @@
     workspaces:
     - name: workspace
       workspace: workspace
+- op: add
+  path: /spec/tasks/-
+  value:
+    name: deprecated-base-image-check
+    runAfter:
+      - sanity-inspect-image
+    taskRef:
+      name: deprecated-image-check
+    params:
+    - name: IMAGE_REGISTRY
+      value: quay.io
```

Comment:
I think we can leave the `IMAGE_REGISTRY` param with the default `registry.access.redhat.com` value, since there should be no entries in Pyxis for `quay.io` images.

Reply:
Good point, fixed